### PR TITLE
libXt, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: x11-libs/libxt/libxt-1.1.5.recipe [hunk 1]
```diff
@@ -0,0 +1,94 @@
+SUMMARY="X Toolkit Intrinsics library"
+DESCRIPTION="libXt provides the X Toolkit Intrinsics, an abstract widget \
+library upon which other toolkits are based. Xt is the basis for many \
+toolkits, including the Athena widgets (Xaw), and LessTif (a Motif \
+implementation)."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
+COPYRIGHT="1993, 1994  X Consortium
```
Comment (alaviss) [inline x11-libs/libxt/libxt-1.1.5.recipe]:
```diff
-COPYRIGHT="1993, 1994  X Consortium
+COPYRIGHT="1993, 1994 X Consortium

File: x11-libs/libxt/libxt-1.1.5.recipe [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	1989, 1998 The Open Group
+	1987, 1988 by Digital Equipment Corporation
+	1993, 2011, Oracle and/or its affiliates
+	2001,2003 Keith Packard"
```
Comment (alaviss) [inline x11-libs/libxt/libxt-1.1.5.recipe]:
```diff
-	2001,2003 Keith Packard"
+	2001, 2003 Keith Packard"

File: x11-libs/libxt/libxt-1.1.5.recipe [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+	autoreconf -vfi
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm -rf $libDir/*.la
```
Comment (alaviss) [inline x11-libs/libxt/libxt-1.1.5.recipe]: They're usually not folders ;)

```diff
-	rm -rf $libDir/*.la
+	rm -f $libDir/*.la

File: x11-libs/libxt/libxt-1.1.5.recipe [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+	libxt$secondaryArchSuffix = $portVersion
+	lib:libXt$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libICE$secondaryArchSuffix
+	lib:libSM$secondaryArchSuffix
+	lib:libX11$secondaryArchSuffix
+	# needed for TEST()
+	lib:libglib_2.0$secondaryArchSuffix
```
Comment (alaviss) [inline x11-libs/libxt/libxt-1.1.5.recipe]: Why does something needed for `TEST()` ends up here?

Comment (Begasus) [reply x11-libs/libxt/libxt-1.1.5.recipe]: Maybe someone will chock on it and write a TEST_requires? Will move it up if you prefer :)

Comment (alaviss) [reply x11-libs/libxt/libxt-1.1.5.recipe]: I mean stuff in `REQUIRES` are not activated during package `TEST()`, only `BUILD_{PRE,}REQUIRES`. If the package doesn't install anything that requires glib2, you can drop this.

Comment (Begasus) [reply x11-libs/libxt/libxt-1.1.5.recipe]: Ah, my bad, will remove

File: x11-libs/libxt/libxt-1.1.5.recipe [hunk 5]
```diff
@@ -0,0 +1,92 @@
+SUMMARY="X Toolkit Intrinsics library"
+DESCRIPTION="libXt provides the X Toolkit Intrinsics, an abstract widget \
+library upon which other toolkits are based. Xt is the basis for many \
+toolkits, including the Athena widgets (Xaw), and LessTif (a Motif \
+implementation)."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
```
Comment (fbrosson) [inline x11-libs/libxt/libxt-1.1.5.recipe]:
```diff
-HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
+HOMEPAGE="https://xorg.freedesktop.org/releases/individual/lib/"
```

File: x11-libs/libxt/libxt-1.1.5.recipe [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+COPYRIGHT="1993, 1994 X Consortium
+	1989, 1998 The Open Group
+	1987, 1988 by Digital Equipment Corporation
+	1993, 2011, Oracle and/or its affiliates
+	2001, 2003 Keith Packard"
+LICENSE="MIT (no promotion)"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXt-$portVersion.tar.gz"
+CHECKSUM_SHA256="b59bee38a9935565fa49dc1bfe84cb30173e2e07e1dcdf801430d4b54eb0caa3"
+SOURCE_FILENAME="libXt-portVersion.tar.gz"
```
Comment (fbrosson) [inline x11-libs/libxt/libxt-1.1.5.recipe]:
```diff
-SOURCE_URI="https://www.x.org/releases/individual/lib/libXt-$portVersion.tar.gz"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXt-$portVersion.tar.bz2"
-CHECKSUM_SHA256="b59bee38a9935565fa49dc1bfe84cb30173e2e07e1dcdf801430d4b54eb0caa3"
+CHECKSUM_SHA256="46eeb6be780211fdd98c5109286618f6707712235fdd19df4ce1e6954f349f1a"
-SOURCE_FILENAME="libXt-portVersion.tar.gz"
```

File: x11-libs/libxt/libxt-1.1.5.recipe (hunk 5 above)
Comment (alaviss) [reply x11-libs/libxt/libxt-1.1.5.recipe]: I think you want this instead, @fbrosson ;)

```diff
-HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
+HOMEPAGE="https://www.x.org/releases/individual/lib/"

File: x11-libs/libxt/libxt-1.1.5.recipe [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+	make install
+
+	rm -f $libDir/*.la
+
+	prepareInstalledDevelLib libXt
+	fixPkgconfig
+
+	# devel package
+	packageEntries devel \
+		$developDir
```
Comment (fbrosson) [inline x11-libs/libxt/libxt-1.1.5.recipe]:
```diff
 	packageEntries devel \
-		$developDir
+		$developDir \
+		$manDir
```
(we only have `$manDir/man3` so we can move the whole `$manDir`)

**Pull request merged**